pitch.md

```diff
@@ -3,42 +3,65 @@
 ## Overview
 _What's this project about, briefly?_
 
+It's an RPG mapmaker language. The initial implementation will be text-based only, with a GUI added only if time permits.
+The language would let users create tile objects by supplying an image source path, specify an optional border, specify a coordinate system and a map size, and then designate rectangular areas to be filled in. Additionally, non-square "tiles" or objects can be placed on the scene by designating an "anchor point" on the input image and then specifying its coordinates on the map. Different images like this can be placed on the scene, so a "tile" may be any image, and the user will also be able to designate layering. Tiles can also be rotated and translated fairly easily. Once a program is run, an output map will be produced.
```

Comment:
If you have a non-square object, you'll need to define the background in some way